docs/infrastructure/eks-quickstart.md

```diff
@@ -33,6 +34,8 @@ Prepare your environment to set up the Amazon EKS cluster:
     -   **main**: Amazon EKS cluster, including the setup of all network and node infrastructure to go from zero to a fully functional cluster
     -   **setup**: Additional setup to be performed before installing Codacy on your vanilla Amazon EKS cluster
 
+    In order to deploy (and eventually destroy) this infrastructure you need to have administration privileges on AWS. The minimum privileges required are listed in the [policy file provided with this documentation](https://raw.githubusercontent.com/codacy/chart/master/docs/infrastructure/EKS/aws-terraform-minimum-admin-policy.json).
```

Comment:
Edit suggestion to make the text more concise:

```suggestion
    You must have administration privileges on AWS to deploy (and eventually destroy) this infrastructure. The policy file [aws-terraform-minimum-admin-policy.json](https://raw.githubusercontent.com/codacy/chart/master/docs/infrastructure/EKS/aws-terraform-minimum-admin-policy.json) lists the minimum privileges that are required.
```

Reply:
Can this .json file be used (perhaps imported?) in AWS to define the necessary privileges? If so, perhaps we could give a hint or include a pointer in this paragraph on how this can be done.

Reply:
I would prefer to maintain it as a guideline. I think most people would just enable admin privileges for their account (or use a privileged account) and won't need to create a specific role with this policy. Those that can't do that most likely have their privileges capped by their devops/it/security/whatever team, which will know what to do with this.

Reply:
OK, makes sense!